### PR TITLE
docs: corrected typo in doc

### LINKED ISSUE #3024: bug: some typo errors in the document
### Issue description

`https://github.com/apache/apisix/blob/master/doc/plugins/request-validation.md`
image
`https://github.com/apache/apisix/blob/master/doc/plugins/http-logger.md`
image

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

fix #3024

### CODE REVIEW

File: doc/plugins/request-validation.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
         "nodes": {
             "127.0.0.1:8080": 1
         }
-    }
+    },
```
Comment (spacewander) [inline doc/plugins/request-validation.md:70]: Changed the wrong place?

**Review by spacewander — CHANGES_REQUESTED** [review]

File: doc/plugins/request-validation.md (hunk 1 above)
Comment (96RadhikaJadhav) [reply doc/plugins/request-validation.md]: @spacewander Apologies. Corrected now :) . Please review.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by juzhiyuan** [comment]
Hi @96RadhikaJadhav, thanks for your nice PR!

Here has one tip: kindly use `fix/feat: #3024` in the first comment to close the target issue once this PR is merged.

**Comment by 96RadhikaJadhav** [comment]
> Hi @96RadhikaJadhav, thanks for your nice PR!
> 
> Here has one tip: kindly use `fix/feat: #3024` in the first comment to close the target issue once this PR is merged.
> 
> image
> 
> Just like this.

Sure :)

**Review by membphis — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**